static/app/views/settings/organizationMembers/organizationMemberRow.tsx

```diff
@@ -230,6 +234,8 @@ export default class OrganizationMemberRow extends PureComponent<Props, State> {
                     ? t(
                         "Your account is managed through your organization's identity provider."
                       )
+                    : isPartnershipUser
+                    ? t('You are a partnership user.')
```

Comment:
We might be able to use the same message here. Like `Partnership users can't be modified` or whatever.

Reply:
yep i would use the same message everywhere